sqldb/sqlerrors.go

```diff
@@ -31,6 +32,26 @@ func MapSQLError(err error) error {
 		return parsePostgresError(pqErr)
 	}
 
+	// Sometimes the error won't be properly wrapped, so we'll need to
+	// inspect raw error itself to detect something we can wrap properly.
+	// This handles a postgres variant of the error.
+	const postgresErrMsg = "could not serialize access"
```

Comment:
Should we create re-usable constants for these error messages so we don't have to declare them twice?